### PR TITLE
Added app accelerator vscode extension docs

### PR DESCRIPTION
**Pull request by DiegoWong**
Which other branches should this be merged with (if any)?

### CODE REVIEW

File: application-accelerator/vscode.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# Dependencies
+To use the vscode extension, you need to interact with the `acc-server`, for more 
+information on how to expose this server follow the instructions [here](./acc-cli.md#server-api-connections)
+
+# Installation
+
+1. Download the file `tanzu-app-accelerator-0.1.0.vsix` from the [release page](https://github.com/pivotal/acc-ide/releases/tag/0.1.0).
+
+1. Open VS Code.
```
Comment (BarrAndrea) [inline application-accelerator/vscode.md:13]: Make this number a 2 please.

**Review by BarrAndrea — CHANGES_REQUESTED** [review]
I have noted a few small changes that are needed to keep in line with our style guide. Great work and thank you for pulling this together.

File: application-accelerator/vscode.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+     
+    1. From the Command Palette (cmd + shift + P), run "Extensions: Install from VSIX...".
+    
+    1. Select the extension file **tanzu-app-accelerator-0.1.0.vsix**.
+    
+    ![Command palette open showing text Extensions: INSTALL FROM VSIX...](../images/vscode-install1.png)
+    
+    **Option 2:**
+    
+    1. Select the **Extensions** tab: ![The extensions tab icon which is a square cut in fourths with the top-right fourth moved away from the other three](../images/vscode-install2.png)
```
[the remark this answers is not in the document]
Comment (DiegoWong) [reply application-accelerator/vscode.md:25]: This is a different option; I think it should be kept as 1

File: application-accelerator/vscode.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    
+    1. Select `Install from VSIX…` from the overflow menu.
```
[the remark this answers is not in the document]
Comment (DiegoWong) [reply application-accelerator/vscode.md:27]: same as previous comment, this should be replaced with a 2 I think

File: application-accelerator/vscode.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+1. Download the file `tanzu-app-accelerator-0.1.0.vsix` from the [release page](https://github.com/pivotal/acc-ide/releases/tag/0.1.0).
+
+1. Open VS Code.
+
+    **Option 1:**
+     
+    1. From the Command Palette (cmd + shift + P), run "Extensions: Install from VSIX...".
+    
+    1. Select the extension file **tanzu-app-accelerator-0.1.0.vsix**.
```
[the remark this answers is not in the document]
Comment (DiegoWong) [reply application-accelerator/vscode.md]: Changed in the latest commit

File: application-accelerator/vscode.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (DiegoWong) [reply application-accelerator/vscode.md]: Changed in the latest commit

File: application-accelerator/vscode.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+# Application Accelerator Visual Studio Code extension
+The application Accelerator Visual Studio Code extension lets you explore and generate projects 
+from the defined accelerators in TAP using VSCode.
```
[the remark this answers is not in the document]
Comment (DiegoWong) [reply application-accelerator/vscode.md]: Changed in the latest commit

File: application-accelerator/vscode.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+# Application Accelerator Visual Studio Code extension
+The application Accelerator Visual Studio Code extension lets you explore and generate projects
```
[the remark this answers is not in the document]
Comment (DiegoWong) [reply application-accelerator/vscode.md:2]: Fixed in the latest commit

**Pull request merged**